.circleci/check-external-links.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 set -e
-
+STATUS=0
 # Select the diff from the branch. Grep the changes indicated with the '+' sign. Use grep with extended regex syntax to identify weblinks. Trim text with sed.
-URLS=`curl -s -L "https://github.com/qlik-oss/core-website/compare/$CIRCLE_BRANCH.diff" | grep '+' | grep -Eo '\(https?://[^ ]+\)' | sed 's/^.\(.*\).$/\1/' | sed 's/)//g' | uniq`
+URLS=$(curl -s -L "https://github.com/qlik-oss/core-website/compare/$CIRCLE_BRANCH.diff" | grep '+' | grep -Eo '\(https?://[^ ]+\)' | sed 's/^.\(.*\).$/\1/' | sed 's/)//g' | uniq)
```

Comment:
As we talked about earlier, `curl` has a flag for following redirects (`-L`) which is used here, so unless the endpoint we are `curl`-ing, it should work and return whatever status code the redirected endpoint sends.

I also checked about custom 404 pages, and they are actually returned with status 404 so we should be good on that front.

Reply:
That's the curl for the diff, I will do the curl to check the link further down, and I'm missing the -L flag there (Update coming up).